packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -49,8 +49,11 @@ contract L1Block is ISemver {
     /// @notice The latest L1 blob base fee.
     uint256 public blobBaseFee;
 
-    /// @custom:semver 1.2.0
-    string public constant version = "1.2.0";
+    /// @notice The chain IDs of the interop dependency set.
+    uint256[] public dependencySet;
+
+    /// @custom:semver 1.2.1
+    string public constant version = "1.2.1";
```

Comment:
Best to update to `1.3.0` with minor version bump because we are adding a new feature that modifies the bytecode